package.json

```diff
@@ -39,6 +39,7 @@
     "angular-ui-bootstrap": "^1.3.3",
     "bootbox": "^5.0.1",
     "bootstrap": "3.4.1",
+    "chromedriver": "89.0.0",
```

Comment:
not test only?

Reply:
We don't have chromedriver.89 in grail repo.  Have to install it via npm and then link to Grails repo